.github/ISSUE_TEMPLATE/new-release-template.md

```diff
@@ -0,0 +1,37 @@
+---
+name: New Release Template
+about: New Release checklist and template
+title: 'chore: tag {version}'
+labels: ''
+assignees: ''
+
+---
+
+Release Checklist
+
+ - [ ] Switch to the main branch.
+ - [ ] `git pull` to ensure the local copy is completely up-to-date.
+ - [ ] `git diff origin/main` to ensure there are no local staged or uncommitted changes.
+ - [ ] Run local testing to ensure no artifacts or other local changes that might break tests have been introduced.
```

Comment:
I think it would be good to link to documentation on how to run the tests locally. 📝